src/Compilers/CSharp/Portable/FlowAnalysis/NullableWalker.cs

```diff
@@ -3256,7 +3242,7 @@ private void AfterLeftChildHasBeenVisited(BoundExpression leftOperand, Conversio
                 var parameters = method.Parameters;
                 visitOperandConversion(binary.Left, leftOperand, leftConversion, parameters[0], leftUnderlyingType);
                 visitOperandConversion(binary.Right, rightOperand, rightConversion, parameters[1], rightUnderlyingType);
-                SetUpdatedSymbol(binary, binary.MethodOpt, method);
+                SetUpdatedSymbol(binary, binary.MethodOpt!, method);
```

Comment:
Should be an assertion

Reply:
We cannot use `method` because that was updated by the call to `AsMemberOfType()` above. But as you said, we've already checked `binary.MethodOpt` indirectly with `method?.ParameterCount == 2`.

---
In reply to: [463271655](https://github.com/dotnet/roslyn/pull/45640#discussion_r463271655) [](ancestors = 463271655)